packages/@aws-cdk/aws-elasticloadbalancing/test/loadbalancer.test.ts

```diff
@@ -89,6 +89,78 @@ describe('tests', () => {
     });
   });
 
+  test('add an Instance as load balancing target', () => {
+    // GIVEN
+    const stack = new Stack();
+    const vpc = new Vpc(stack, 'VCP');
+    const securityGroup = new SecurityGroup(stack, 'simple-instance-1-sg',
+      {
+        vpc,
+        allowAllOutbound: true, // will let your instance send outboud traffic
+        securityGroupName: 'simple-instance-1-sg',
+      },
+    );
+
+    // lets use the security group to allow inbound traffic on specific ports
+    securityGroup.addIngressRule(
+      Peer.anyIpv4(),
+      Port.tcp(22),
+      'Allows SSH access from Internet',
+    );
+
+    securityGroup.addIngressRule(
+      Peer.anyIpv4(),
+      Port.tcp(80),
+      'Allows HTTP access from Internet',
+    );
+
+    securityGroup.addIngressRule(
+      Peer.anyIpv4(),
+      Port.tcp(443),
+      'Allows HTTPS access from Internet',
+    );
```

Comment:
This looks like it's testing things about `SecurityGroup`, but we're not actually testing SecurityGroup here.